install/install-serverless-logic-operator.adoc

```diff
@@ -0,0 +1,21 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="install-serverless-logic-operator"]
+= Installing the {ServerlessLogicOperatorName}
+:context: install-serverless-logic-operator
+
+toc::[]
+
+Installing the {ServerlessLogicOperatorName} enables you to install and use Knative components on a {ocp-product-title} cluster. The {ServerlessLogicOperatorName} manages Knative custom resource definitions (CRDs) for your cluster and enables you to configure them without directly modifying individual config maps for each component.
```

Comment:
```suggestion
Installing the {ServerlessLogicOperatorName} enables you to install and use serverless workflows on a {ocp-product-title} cluster. The {ServerlessLogicOperatorName} manages Kubernetes custom resource definitions (CRDs) to configure and deploy the workflows.
```

Reply:
wdyt? @domhanak

Reply:
Please add a link to the `https://serverlessworkflow.io` in `serverless workflows`.

Reply:
Added